src/lib/reminderFields.ts

```diff
@@ -26,6 +29,16 @@ export const buildReminderFields = (today: Date = new Date()): ReminderFields =>
     };
 };
 
-export const getReminderFields = (reminderFields?: ReminderFields): ReminderFields => {
-    return !!reminderFields ? reminderFields : buildReminderFields();
+export const getReminderFields = (
+    variant: Variant,
+    targeting: EpicTargeting,
+): ReminderFields | undefined => {
+    const userCohorts = getUserCohorts(targeting);
+    const isSupporter = userCohorts.includes('AllExistingSupporters');
+
+    if (isSupporter) {
+        return undefined;
+    }
+
+    return !!variant.showReminderFields ? variant.showReminderFields : buildReminderFields();
```

Comment:
I think we will probably be able to remove this at some point, when we allow the reminder to be customised in the tooling. That shouldn't be too tricky. We can update the `ContributionsReminderSecondaryCta` to something like:

```ts
interface ContributionsReminderSecondaryCta {
    type: SecondaryCtaType.ContributionsReminder;
    reminder: Reminder;
}
```